README.md

```diff
@@ -0,0 +1,76 @@
+DeDuplicated Queue MySQL Plugin
+===============================
+
+[![Build Status][travis-image]][Travis CI]
+[![Dependencies][dependencies-image]][Dependencies]
+[![Dev Dependencies][devdependencies-image]][Dev Dependencies]
+[![codecov.io][codecov-image]][Code Coverage]
+
+
+About
+-----
+
+A MySQL plugin for the [DeDuplicated Queue (DDQ)](https://www.npmjs.com/package/ddq) module.
+
+
+Configuration
+-------------
+
+In order to get up and running, you'll need to fill out several config options. DDQ instantiates the DDQ MySQL Plugin and passes in a config file, so do not create another config file for the DDQ MySQL Plugin. Simply add these fields to your DDQ config and they will be utilized by the Plugin.
+
+Here's an example of a DDQ config that would work with this plugin:
+
+
+    {
+        backend: "MySQL"
+        backendConfig: {
+            database: "exampleDatabase",
+            heartbeatCleanupDelayMs: 3000
+            heartbeatLifetimeSeconds: 3,
+            host: "localhost",
+            password: "examplePassword",
+            pollingDelayMs: 5000,
+            port: 3306,
+            table: 'exampleTable',
+            topics: [
+                "topic1",
+                "topic2",
+                null
+            ],
+            user: "exampleUser",
+        },
+        heartbeatDelayMs: 1000,
+        createMessageCycleLimit: 10
```

Comment:
This is not JSON.  It is a JS object literal.  For it to be JSON you need to quote properties.